virt/virt-4-9-release-notes.adoc

```diff
@@ -23,6 +23,8 @@ include::modules/virt-supported-cluster-version.adoc[leveloffset=+2]
 {VirtProductName} guests can use the following operating systems:
 
 * Red Hat Enterprise Linux 6, 7, and 8.
+* Red Hat Enterprise Linux 9 Alpha (TP).
+* Fedora 31 and 32.
```

Comment:
We support Fedora 34 
annotations:
defaults.template.kubevirt.io/disk: rootdisk
description: 'Template for Fedora 32 VM or newer. A PVC with the Fedora disk image
must be available. Recommended disk image: https://download.fedoraproject.org/pub/fedora/linux/releases/32/Cloud/x86_64/images/Fedora-Cloud-Base-32-1.6.x86_64.qcow2'
iconClass: icon-fedora
name.os.template.kubevirt.io/fedora32: Fedora 32 or higher
name.os.template.kubevirt.io/fedora33: Fedora 32 or higher
name.os.template.kubevirt.io/fedora34: Fedora 32 or higher
name.os.template.kubevirt.io/silverblue32: Fedora 32 or higher
name.os.template.kubevirt.io/silverblue33: Fedora 32 or higher
name.os.template.kubevirt.io/silverblue34: Fedora 32 or higher
openshift.io/display-name: Fedora 32+ VM
openshift.io/documentation-url: https://github.com/kubevirt/common-templates
openshift.io/provider-display-name: KubeVirt
openshift.io/support-url: https://github.com/kubevirt/common-templates/issues
operator-sdk/primary-resource: openshift-cnv/ssp-kubevirt-hyperconverged
operator-sdk/primary-resource-type: SSP.ssp.kubevirt.io
tags: hidden,kubevirt,virtualmachine,fedora

Reply:
After checking with PM. we should not mention Fedora as supported OS. please remove it.